### DISCUSSION TITLE
Maybe the INFORMATION_SCHEMA can be lower case?

### DISCUSSION THREAD

**Discussion by TCeason** [databendlabs/databend#8515] [General]
https://github.com/datafuselabs/databend/issues/4980

We can see the dbeaver will send query : 

```
SELECT * FROM information_schema.TABLES t\nWHERE\n\tt.TABLE_SCHEMA = 'information_schema'\n\tAND t.TABLE_NAME = 'CHECK_CONSTRAINTS'
```

Now we db name and table name in `information_schema` are uppper case. But the columns in `information_schema` is lowwer case.

Now in planner v2 we has already support 

```
| unquoted_ident_case_sensitive  | 0          | 0          | SESSION | Case sensitivity of unquoted identifiers, default value: 0 (aka case-insensitive).                  | UInt64 |
```

So maybe the db name and table name can set to lowwer case?

**Comment by TCeason** [comment]
@sundy-li

**Reply by sundy-li** [reply]
ok
